### ISSUE TITLE
HTML Sanitisation Fix

### ISSUE THREAD

**Issue by Daxorinator** [ugcompsoc/compsoc.ie#23]
* Fix WIP HTML Sanitizer and implement on the site to fix HTML sanitization issue with events on the front page

**Comment by Daxorinator** [comment]
@addmmyyyy @pan-grayza did ye get this sorted?

**Comment by pan-grayza** [comment]
My ssd broke, this Wednesday so I will be able to do work only after next weekend. Waiting for new one rn. Discord is lagging for me so I couldn't inform.

**Comment by Daxorinator** [comment]
No longer needed due to major website overhaul
